### PR TITLE
Ik/single entry point

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: ote_cli/ote_cli/registry/registry.py [hunk 1]
```diff
@@ -52,3 +56,14 @@ def get(self, name):
 
     def __repr__(self):
         return yaml.dump(self.templates)
+
+
+def find_and_parse_model_template(path_or_name):
+    if os.path.exists(path_or_name):
+        return parse_model_template(path_or_name)
+    template = [template for template in Registry('.').templates if template['name'] == path_or_name]
+    if template:
```
Comment (druzhkov-paul) [inline ote_cli/ote_cli/registry/registry.py:65]: What if multiple matching templates are found?

Comment (Ilya-Krylov) [reply ote_cli/ote_cli/registry/registry.py:65]: It must not be so, right?

Comment (druzhkov-paul) [reply ote_cli/ote_cli/registry/registry.py:65]: If name == model_template_id, then yes. ID is unique, but not the name.

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: ote_cli/ote_cli/registry/registry.py (hunk 1 above)
Comment (Ilya-Krylov) [reply ote_cli/ote_cli/registry/registry.py]: ok, then path and id can be used only.

**Review by druzhkov-paul — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
bmm 19072 🕐

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Review by druzhkov-paul — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
19120 ✔️

**Pull request merged**